services/web/server/src/simcore_service_webserver/projects/_metadata_handlers.py

```diff
@@ -88,11 +94,15 @@ async def update_project_metadata(request: web.Request) -> web.Response:
     path_params = parse_request_path_parameters_as(ProjectPathParams, request)
     update = await parse_request_body_as(ProjectMetadataUpdate, request)
 
-    custom_metadata = await _metadata_api.set_project_custom_metadata(
+    custom_metadata = await _metadata_api.set_project_metadata(
         request.app,
         user_id=req_ctx.user_id,
         project_uuid=path_params.project_id,
         value=update.custom,
+        # NOTE: MB this is where the PublicAPI shall bring in the parent node
+        # see https://github.com/ITISFoundation/osparc-simcore/issues/5816
+        parent_project_uuid=None,
+        parent_node_id=None,
```

Comment:
@sanderegg one thing confuses me here: Why are these values set to `None`? Shouldn't they be taken from the headers/body/query params? Or will that be done in a second step?

Reply:
I would suggest to already add these params here so that I can proceed to pass these headers via the public api.

Reply:
ok will do